src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -151,7 +181,7 @@ export class HotbarMenu extends React.Component<Props> {
     return (
       <div className={cssNames("HotbarMenu flex column", className)}>
         <div className="HotbarItems flex column gaps">
-          <DragDropContext onDragEnd={this.onDragEnd}>
+          <DragDropContext onDragEnd={this.onDragEnd.bind(this)}>
```

Comment:
nit: I'd better to use arrow-function for all event-handlers by default (or `@boundMethod`, but it's would be available only after merging `mobx6` migration branch)